### DISCUSSION TITLE
A courtesy request for Rush_blade_F7_HD～～～

### DISCUSSION THREAD

**Discussion by elfox-14482** [iNavFlight/inav#8072] [Q&A]
HI guys, im big fan of inav. glad to see the 5.0 Pre-release

Unfortunately, I didn't find the firmware for "Rush_blade_F7_HD" in 5.0 again~~.  in my flight club, there many users had this flight control. so anyone can kindly provide this firmware for 5.0?

and also strongly suggest add the Rush_blade_F7_HD to the future inav version.

Thx~

**Comment by MrD-RC** [comment]
Contact Rush and let them know that you would like them to support iNav on their flight controllers. 

https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
